fiftyone/utils/fiw.py

```
@@ -283,51 +286,331 @@ def _parse_identifier(identifier):
     return "/".join(id_parts[:2])
 
 
-def download_fiw_dataset(dataset_dir, split, scratch_dir=None, cleanup=False):
+def parse_fiw_dataset(source_dir, dataset_dir, scratch_dir, split):
```

Ruff (0.14.8)

Unused function argument: `scratch_dir`

(ARG001)